src/views/DatasetManager/components/DatasetMetadata/DatasetMetadata.js

```diff
@@ -23,9 +23,11 @@ const useStyles = makeStyles((theme) => ({
 export const DatasetMetadata = () => {
   const classes = useStyles();
   const { t } = useTranslation();
-  const { dataset } = useDatasetMetadata();
+  const { dataset, updateDataset } = useDatasetMetadata();
+  const datasetId = dataset?.id;
+  const datasetTags = dataset?.tags;
```

Comment:
`datasetTags` is only used once below, do we really need to declare this const ?